helm/polaris/README.md.gotmpl

```diff
@@ -101,27 +95,24 @@ The below instructions assume a local Kubernetes cluster is running and Helm is
 
 #### Common setup
 
-Create and populate the target namespace:
+Create the target namespace:
 
 ```bash
 kubectl create namespace polaris
-kubectl apply --namespace polaris -f helm/polaris/ci/fixtures/
-
-kubectl wait --namespace polaris --for=condition=ready pod --selector=app.kubernetes.io/name=postgres --timeout=120s
 ```
 
-The `helm/polaris/ci` contains a number of values files that can be used to install the chart with
-different configurations.
-
-You can also run `ct` (chart-testing):
-
-```bash
-ct lint --charts helm/polaris
-ct install --namespace polaris --debug --charts ./helm/polaris
-```
+Create all the required resources in the `polaris` namespace. This usually includes a Postgres
```

Comment:
I'm a little doubtful about this part of the change. I'm not sure if it's wrong, but it does feel like a slight regression. Could we just keep creating these resources?